### ISSUE TITLE
ON- HOLD Restructure the WASM deploy tool to be used as CLI tool

### ISSUE THREAD

**Issue by prayagd** [pendulum-chain/wasm-deploy#10]
## Context 
Ticket related to make the WASM deploy tool usable to for builders other than Nabla.

## Requirement
Currently wasm-deploy contains a sample project in its own repository and works only with this sample project. Instead of this wasm-deploy should be a standalone tool that is agnostic of a particular project and can should be usable for any other npm project.

Wasm-deploy should contain a bin tool so that after adding it as a dependency in a 3rd party project it should be possible to add an npm script of the form `npm wasm-deploy` to that 3rd party project.

## On-Hold
Waiting for [this](https://github.com/0xamberhq/contracts/issues/73) issue to be completed, which will move the Nabla deployment scripts to the corresponding Nabla repository.

**Comment by prayagd** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/wasm-deploy/10?planning-poker) with Zenhub @ebma @TorstenStueber

**Comment by prayagd** [comment]
@gianfra-t what stopping this ticket from getting closed? Is it already reviewed?

**Comment by TorstenStueber** [comment]
It's on hold: https://github.com/pendulum-chain/wasm-deploy/pull/24#issuecomment-1856642720

**Comment by prayagd** [comment]
Thanks for informing will add that label as on-hold

**Comment by prayagd** [comment]
@gianfra-t whats the latest update here? still on hold?

**Comment by gianfra-t** [comment]
@prayagd Still on hold. And it will require some re-work once the mentioned blocking issue is finished.
